### PR TITLE
Get list of objects type mapping error handling in "execution" phase

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Part of #396.

@nikitin-da PTAL!

### CODE REVIEW

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/get/PreparedGetListOfObjects.java [hunk 1]
```diff
@@ -55,6 +64,22 @@
...
+
+        if (explicitGetResolver != null) {
+            getResolver = explicitGetResolver;
+        } else {
+            final ContentResolverTypeMapping<T> typeMapping = storIOContentResolver.internal().typeMapping(type);
+
+            if (typeMapping == null) {
+                throw new IllegalStateException("This type does not have type mapping: " +
+                        "type = " + type + "," +
+                        "ContentProvider was not affected by this operation, please add type mapping for this type");
```
Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/get/PreparedGetListOfObjects.java]: Could we affect `contentProvider` with query?

Comment (artem-zinnatullin) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/get/PreparedGetListOfObjects.java]: I'll change to "ContentProvider was not touched by this operation", okay?

Comment (nikitin-da) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operation/get/PreparedGetListOfObjects.java]: Ok))

**Comment by artem-zinnatullin** [comment]
Please comment if this PR LGTY, I'll rebase it and squash into one commit!

**Comment by nikitin-da** [comment]
PR LGTM

**Comment by artem-zinnatullin** [comment]
Squashed into one commit!

**Comment by nikitin-da** [comment]
Super! Waining for ci...

**Pull request merged**